pages/common/dash.md

```diff
@@ -0,0 +1,20 @@
+# dash
+
+> Debian Almquist Shell, a modern, POSIX-compliant implementation of `sh` (not Bash-compatible).
+> More information: <https://manned.org/dash>.
+
+- Start an interactive shell session:
+
+`dash`
+
+- Execute a command and then exit:
+
+`dash -c "{{command}}"`
+
+- Execute a script:
+
+`dash {{path/to/script.sh}}`
+
+- Run commands from a script, printing each command before executing it:
+
+`dash -x {{path/to/script.sh}}`
```

Comment:
```

- Execute commands from a script, stopping at the first error:

`dash -e {{path/to/script.sh}}`

- Execute commands from stdin:

`dash -s`
```

Reply:
May be we can add the same samples as in bash with the `-s` and `-e` flags as they are the same?

Reply:
arf I'm sorry I added my comments after the merge. Nevermind.

Reply:
If it works, by all means.  I'll do it later today 👍🏻

Reply:
Sounds good to me. I suggest `Read and execute` though on that last example though.